addons/udes_stock/models/stock_picking_type.py

```diff
@@ -2,9 +2,43 @@
 from odoo import fields, models
 
 
+TARGET_STORAGE_FORMAT_OPTIONS = [
+    ("pallet_products", "Pallet of products"),
+    ("pallet_packages", "Pallet of packages"),
+    ("package", "Packages"),
+    ("product", "Products"),
+]
+
+
 class StockPickingType(models.Model):
     _inherit = "stock.picking.type"
 
+    u_user_scans = fields.Selection(
+        [("pallet", "Pallets"), ("package", "Packages"), ("product", "Products")],
+        string="What the User Scans",
+        help="What the user scans when asked to " "scan something from pickings of this type",
```

Comment:
Don't need the extra quotes in the middle, this was probably a black formatting thing in UDES 11 originally.